### PR TITLE
Update to Valet 4 driver syntax

### PR DESCRIPTION
**Pull request by drbyte**

### CODE REVIEW

File: DistValetDriver.php [hunk 1]
```diff
@@ -39,7 +43,10 @@ public function isStaticFile($sitePath, $siteName, $uri)
      * @param  string  $uri
      * @return string
      */
-    public function frontControllerPath($sitePath, $siteName, $uri)
+    /**
```
Comment (DCzajkowski) [inline DistValetDriver.php]: There are two docblocks for this method now. There should be one

Comment (drbyte) [reply DistValetDriver.php]: Oops. Fixed.

File: DistValetDriver.php [hunk 2]
```diff
@@ -23,7 +27,7 @@ public function serves($sitePath, $siteName, $uri)
      * @param  string  $uri
      * @return string|false
      */
-    public function isStaticFile($sitePath, $siteName, $uri)
+    public function isStaticFile(string $sitePath, string $siteName, string $uri)/*: string|false */
```
Comment (DCzajkowski) [inline DistValetDriver.php:30]: Sorry it took me so long. Should this return statement be a comment?

Comment (drbyte) [reply DistValetDriver.php:30]: yes, in Valet 4 the return-type is commented out by default in order to not crash on older PHP versions. At some point when older versions fall away this won't be an issue any longer, but this is consistent with Valet internals for now.

Comment (DCzajkowski) [reply DistValetDriver.php:30]: Thank you for the explanation!

**Comment by DCzajkowski** [comment]
Thanks for your contribution. I don't use Valet anymore, but this will be useful to the countless community members. Have a great day! 😊

**Pull request merged**